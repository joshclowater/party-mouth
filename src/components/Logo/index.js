import styled, { keyframes } from 'styled-components';

import logo from 'src/assets/logo.svg';

const rotate360 = keyframes`
  from { transform: rotate(0deg); }
  to { transform: rotate(360deg); }
`;

export default styled.img.attrs({
  src: logo,
  alt: 'logo'
})`
  animation: ${rotate360} infinite 20s linear;
  height: 12vh;
`;
