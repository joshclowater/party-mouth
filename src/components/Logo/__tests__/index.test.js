import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import Logo from '../';

it('Renders correctly', () => {
  expect(shallow(<Logo />)).toMatchSnapshot();
});
