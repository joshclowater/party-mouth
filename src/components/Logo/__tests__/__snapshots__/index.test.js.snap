// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
.c0 {
  -webkit-animation: iVXCSc infinite 20s linear;
  animation: iVXCSc infinite 20s linear;
  height: 12vh;
}

<img
  alt="logo"
  className="c0"
  src="logo.svg"
/>
`;
